DistFiles/xMatter/bloom-xmatter-mixins.jade

```diff
@@ -146,12 +146,8 @@ mixin factoryStandard-titlePage
 		#languageInformation
 			.languagesOfBook(data-book='languagesOfBook')
 			//- review: can we get rid of these "langName" classes?
```

Comment:
Right. I didn't even add that comment, it was there previously. (Probably you wrote it :-)